corehq/apps/case_importer/do_import.py

```diff
@@ -110,29 +110,15 @@ def user(self):
         return CouchUser.get_by_user_id(self.config.couch_user_id, self.domain)
 
     @cached_property
-    def location_ids_accessible_to_user(self):
+    def locations_accessible_to_user(self):
```

Comment:
I am knowingly returning two different data types, and then passing them to `_CaseImportRow` in same parameter. This doesn't feel great, but passing yet another parameter felt even worse. Using a constant seemed better than using `None`.

Reply:
Makes sense.  If you really don't like it, another approach would be to pass a `is_accessible_to_user(location)` callable.  We don't often use first-class-functions in HQ code, so I'd also understand if you find that even more confusing.

```python
def is_accessible_to_user(self, location_id):
    return (
        self.user.has_permission(self.domain, 'access_all_locations')
        or location_id in self._locations_accessible_to_user
    )

@cached_property
def locations_accessible_to_user(self):
    ...
```